libs/ui/widgets/src/lib/popover/popover.component.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  Component,
+  AfterViewInit,
+  ElementRef,
+  Input,
+  ViewChild,
+  OnDestroy,
+  OnChanges,
+  SimpleChanges,
+  TemplateRef,
+  Renderer2,
+  ViewContainerRef,
+  EmbeddedViewRef,
+} from '@angular/core'
+import tippy, { Instance } from 'tippy.js'
+
+@Component({
+  selector: 'gn-ui-popover',
+  templateUrl: './popover.component.html',
+  styleUrls: ['./popover.component.css'],
+})
+export class PopoverComponent implements AfterViewInit, OnChanges, OnDestroy {
+  @ViewChild('popoverContent', { static: false }) popoverContent: ElementRef
+  @Input() content: string | TemplateRef<any>
+  @Input() theme: 'light' | 'light-border' | 'translucent' | 'material' | ''
```

Comment:
Somehow the theming doesn't seem to get applied. Not in the storybook nor in the datahub. Is there something I'm missing?

Reply:
my screenshoot come from the story-book using light-border. Your screenshoot looks like the default theming same as ''.  I don't know yet why it doesn't apply on your side. Which browser do you use?
I had added only light-border on datahub and the differents on storybook.

Reply:
Now the theming works on my side as well. Not sure what was the issue. Never mind! Looks good to me.